app/workers/search_for_external_registry_bikes_worker.rb

```diff
@@ -0,0 +1,28 @@
+class SearchForExternalRegistryBikesWorker < ScheduledWorker
+  prepend ScheduledWorkerRecorder
+
+  def self.frequency
+    24.hours
+  end
+
+  def perform
+    search_dutch_impounded_registries
+  end
+
+  # Search external registries. Results are persisted as ExternalRegistryBike records
+  def search_dutch_impounded_registries
```

Comment:
Please create separate sidekiq jobs for each serial. Sidekiq should manage the parallelism.

One pattern being used is to run a single search when the job is passed an argument, and enqueue the running of the jobs if no argument is passed.